docs/ca/dev/preparation-for-going-live.md

```diff
@@ -23,191 +23,108 @@ We've divided the preparation into approximate timeframes, and you can adjust th
+| CLO-12 | Cloud       | Prepare a DNS strategy                                   | Decide how you want customers to access your shop and verify that you control access and can manage the DNS. For example, the domain of your shop is `spryker.com`, but you want customers to access the Storefront at `www.spryker.com`. For details on how to set up DNS, see [Set up DNS](/docs/ca/dev/set-up-dns.html).<br>- Optional: Delegate DNS. To find out how to delegate a DNS name, see [Setting up a custom SSL certificate](/docs/ca/dev/setting-up-a-custom-ssl-certificate.html).<br>- Make sure Transport Layer security (TLS) certificates are provisioned. If you delegate DNS to Spryker, TLS certificates for your endpoints are created automatically. If you want us to create TLS certificates for your endpoints but don't want to delegate your DNS, request the verification of DNS records through the [Support Portal](https://support.spryker.com). If you don't delegate your DNS and want to use your own certificates, provide them to us as described in [Setting up a custom SSL certificate](/docs/ca/dev/setting-up-a-custom-ssl-certificate.html). |                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
[Setting up a custom SSL certificate](/docs/ca/dev/setting-up-a-custom-ssl-certificate.html) doesn't explain how to delegate DNS

Reply:
Fair point. 
Lets keep it as it was in the previous version and I will try to update it in the next release